package.json

```diff
@@ -97,7 +97,7 @@
     "bunyan-format": "^0.2.1",
     "compression": "^1.7.5",
     "connect-flash": "^0.1.1",
-    "connect-redis": "^7.1.1",
+    "connect-redis": "^8.0.1",
```

Comment:
the docker compose file specifies a 7.* version, needs fixing too

Reply:
ok so prod uses 7.2 but the template requies connect-redis 8 which is a bit of a discrepancy. let's leave docker-compose as it is